GitHub-Test-4-PyNLPDiscuss/Milo-GitTest-4.md

```diff
@@ -0,0 +1,19 @@
+# Git Test Discussion Formatting
+
+Before we start, I forgot to put a codeblock in my last mardown file, so here is a code block:
+```
+Some code can go here
+```
+## Questions 
+1. Bigrams and Trigrams Interest
+     - Bigrams and Trigrams really stuck out to me as interesting forms of modeling because it gives the concept of NLP a more visualized sense, which makes it easier to understand for me. It reminded me of vendiagrams that we would do in school, however it is just for a word rather than topics. I am just curious how computers would read these. I understand there is a part about overlapping data with the words, but how? Does it access a database of word meaning and hundreds of instances of word usage to understand what a particular instance of a specific word means?
+2. One To Work With
+     - Bigrams a Trigrams really are the ones I would like the work with because of the visualization aspect. I can actually visualize how words are related and link libraries of word usage to them. The other ones I just have a hard time wrapping my brain around, this one just makes the most sense to me.
```

Comment:
I also find this to be very interesting because, I am more of a visual learner. I like to see things laid out.